### PR TITLE
a few updates and fixes to the readme

### PR DESCRIPTION
**Pull request by coreydaley**

### CODE REVIEW

**Comment by coreydaley** [comment]
@hhorak ptal

File: README.md [hunk 1]
```diff
@@ -8,38 +8,40 @@ Users can choose between RHEL and CentOS based builder images.
...
 
 For more information about using these images with OpenShift, please see the
-official [OpenShift Documentation](https://docs.openshift.org/latest/using_images/s2i_images/perl.html).
+official [OpenShift Origin documentation](https://docs.openshift.org/latest/using_images/s2i_images/perl.html).
 
 Versions
 ---------------
-Perl versions currently provided are:
+Perl versions currently provided:
+* perl-5.16
```
Comment (bparees) [inline README.md]: 5.16 is deprecated (the scl packages are) so that's why we don't advertise this one.

File: README.md [hunk 2]
```diff
@@ -60,10 +62,13 @@ on all provided versions of Perl.**
 Usage
 ---------------------------------
 
-For information about usage of Dockerfile for Perl 5.20,
+For information about usage of the Dockerfile for Perl 5.16,
+see [usage documentation](5.16/README.md)
```
Comment (bparees) [inline README.md]: same here

**Comment by coreydaley** [comment]
@bparees I find it confusing that the 5.16 folder exists and is not mentioned anywhere.  Maybe it should be removed if it is deprecated?

**Comment by bparees** [comment]
The 5.16 folder does have a readme stating that it's been deprecated.

I prefer keeping the folder for historical reasons.  It's unsupported, but someone who really wants to could still build it.

But if that's not sufficient in your mind, i'd be ok w/ you listing 5.16 as long as you state that it's deprecated.

**Comment by coreydaley** [comment]
@bparees updated

**Comment by bparees** [comment]
Lgtm

Ben Parees | OpenShift

**Comment by coreydaley** [comment]
[test]

**Comment by bparees** [comment]
[test]

**Comment by bparees** [comment]
looks like i'm only an admin for the centos tests, sorry @coreydaley

**Comment by hhorak** [comment]
[test]

**Comment by hhorak** [comment]
CentOS testing environment is somehow broken, but since this is just a change in main README, lgtm. Thanks!

**Pull request merged**